test/jdk/jdk/crac/SecureRandom/ReseedTest.java

```diff
@@ -70,7 +71,7 @@ public void exec() throws Exception {
             throw new RuntimeException("Restore ERROR " + e);
         }
 
-        int r = sr.nextInt(255);
-        System.exit(r);
+        System.out.println(sr.nextInt());
+        System.exit(0);
```

Comment:
This change will also help when something is wrong with the restore, the previous version might confuse that situation with legitimate return with non-zero error. 

On the second thought, could you extract this into a separate PR?

Reply:
If you promise me that you'll integrate right away, I can :)

Reply:
Here it goes: https://github.com/openjdk/crac/pull/55
I've reverted the commit in this PR